src/include/network/lan/complex.rb

```diff
@@ -393,14 +446,23 @@ def handleOverview(_key, event)
           # As it could easily happen that all interfaces are set to "no" (and
           # default route is unrecheable in such case) this explicite setup was
           # added.
+          # FIXME: not implemented in network-ng
           LanItems.set_default_route = true
 
           return :add
         when :edit
-          if LanItems.IsCurrentConfigured
-            LanItems.SetItem
-
-            if LanItems.startmode == "managed"
+          iface = config.interfaces.find(iface_name)
+          # FIXME: turn following lines into something like builder.load(iface) ?
+          @builder.name = iface.name
+          @builder.type = iface.type
+          if iface.configured
+            @builder.load_sysconfig(iface.config)
```

Comment:
again too much tight to sysconfig. I prefer here generic load. In general I do not like naming here, as it is more like load current config to builder. Not anything about sysconfig.

Reply:
yes it is. Bcs I'm currently injecting new things into old workflow. And as i currently use sysconfig option names as identifiers in the builder I've picked this name. Definitely something for improving